lib/spack/spack/tengine.py

```diff
@@ -11,6 +11,7 @@
 import llnl.util.lang
 
 import spack.config
+import spack.extensions
```

Comment:
Is this vestigial?

Reply:
I noticed it was a missing import -- I can drop it from this PR to keep the squash commit on develop clean if you want

Reply:
Nah, if it's missing leave it.